### PR TITLE
ST: New tag for ST

### PR DESCRIPTION
**Pull request by Frawless**
### Type of change

- Bugfix
- Enhancement / new feature

### Description

Added new tag for tests - test which are not 100% stable are marked as FLAKY now. 

I found one issue with HelmChartST - guess no need to create separate PR for this.

Positions of timeouts and polls intervals were incorrect so I corrected them in this PR, because it will be merged before [`PR#1248`](https://github.com/strimzi/strimzi-kafka-operator/pull/1248). 

### Checklist

- [ ] Make sure all tests pass
- [ ] Update documentation

### CODE REVIEW

File: test/src/main/java/io/strimzi/test/extensions/StrimziExtension.java [hunk 1]
```diff
@@ -96,6 +96,7 @@
     /** Tags */
     public static final String ACCEPTANCE = "acceptance";
     public static final String REGRESSION = "regression";
+    public static final String FLAKY = "flaky";
```
Comment (tombentley) [inline test/src/main/java/io/strimzi/test/extensions/StrimziExtension.java]: We should document each tag to say what the intended semantics are

Comment (Frawless) [reply test/src/main/java/io/strimzi/test/extensions/StrimziExtension.java]: Done

File: test/src/main/java/io/strimzi/test/extensions/StrimziExtension.java [hunk 2]
```diff
@@ -93,9 +93,18 @@
...
+     */
     public static final String ACCEPTANCE = "acceptance";
+    /**
+     * Tag for regression tests which are stable.
+     */
     public static final String REGRESSION = "regression";
+    /**
+     * Tag for tests, which's results are not 100% reliable on all testing environments.
+     */
+    public static final String FLAKY = "flaky";
```
Comment (tombentley) [inline test/src/main/java/io/strimzi/test/extensions/StrimziExtension.java]: So would be be useful to split this up according to the test environment, allowing us to only exclude flaky tests on the environment where they were flaky. That would increase coverage on platforms where those tests were not flaky, which must be a good thing, right?

**Comment by tombentley** [comment]
Right

**Pull request merged**